coala_quickstart/interaction/Logo.py

```diff
@@ -0,0 +1,118 @@
+from coala_quickstart.Strings import coala_bear_logo, welcome_messages
+
+
+def text_wrap(*args, delimiter=' ', limit=80):
```

Comment:
so @SanketDG has blogged about this textwrap thing and it seems like it's the same, @SanketDG mind commenting here?